src/DatabaseGpt.Npgsql/NpgsqlDatabaseGptProvider.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Data;
+using System.Text;
+using Dapper;
+using DatabaseGpt.Abstractions;
+using DatabaseGpt.Abstractions.Exceptions;
+using Npgsql;
+
+namespace DatabaseGpt.Npgsql;
+
+public class NpgsqlDatabaseGptProvider : IDatabaseGptProvider, IDisposable
+{
+    private readonly NpgsqlDatabaseGptProviderConfiguration settings;
+    private bool disposedValue;
+
+    public NpgsqlDatabaseGptProvider(NpgsqlDatabaseGptProviderConfiguration settings)
+    {
+        this.settings = settings;
+        this.connection = new NpgsqlConnection(settings.ConnectionString);
+        connection.Open();
+    }
+
+    private readonly NpgsqlConnection connection;
+
+    public string Name => "Postgres";
+    public string Language => "Postgres SQL";
```

Comment:
Place these properties on top, after the `private` fields. Place a blank line between the properties.